experiment/methods/operon/regressor.py

```diff
@@ -113,7 +113,7 @@ def callback(self, study, trial):
 
 
 class OperonOptuna(BaseEstimator, RegressorMixin):
-    def __init__(self, name, n_folds=3, n_trials=100, max_time=3600, fixed_params={}, suggest_callback=None, random_state=None):
+    def __init__(self, name, n_folds=3, n_trials=100, max_time=1,0, fixed_params={}, suggest_callback=None, random_state=None):
```

Comment:
Are you trying to set max_time to 1 second? Is this right, a `.` instead of `,`?

Reply:
Oops, that was a global search-and-replace gone wrong. Reverted it.